wordpress-mobile/WordPress-iOS
# Add verifcation prompt when unverified user tries to publish a post

👋 , long time listener, first time caller here.

This PR improves the experience for unverified users trying to publish a post. Here's how it looks like:

![simulator screen shot - iphone 8 plus - 2017-10-04 at 16 48 29](https://user-images.githubusercontent.com/1594081/31198394-6dfc6d1e-a954-11e7-9400-65968b5a6209.png)

I diverged slightly from the designs — since the "Resend" email action hits the network, I felt like using a "regular" button was better, since we could indicate to the users that a request is in-flight.

(One problem: "Resend email" doesn't fit on that button, so I shortened it to "Resend", but I feel like it's not obvious what it does now. I'm sure there's a better way to phrase that. In general, I'd be super happy for better copy suggestions here.)

Fixes #6035.

## To test:

1. Create an a new WPCom account from the website.
2. Don't verify the email.
3. Try to publish a post from the app.
